src/Raven.Studio/typescript/viewmodels/shell/registration.ts

```diff
@@ -88,45 +91,59 @@ class registration extends dialogViewModelBase {
             const now = moment();
             return firstStart.diff(now, "days");
         });
+
+        let url = "https://ravendb.net/request-license?";
```

Comment:
code duplication - can we move this to licnese model ?

Reply:
done